lib/domodule.js

```diff
@@ -169,4 +169,20 @@ export default class Domodule { // eslint-disable-line no-unused-vars
       }
     }
   }
+
+  hide(el = this.el) {
+    if (!(el instanceof Node)) {
+      el = this.find(el);
+    }
+
+    el.style.display = 'none';
```

Comment:
Yeah we can do that, however that won't always work as `show` may be used to reveal an element hidden by CSS. I think the best bet is to just remove the style tag from the element if present and, otherwise, use whatever is provided or `block` by default. This is better if you start to mix MQ which may have different values on different sizes.